kolibri/core/auth/tasks.py

```diff
@@ -316,15 +316,15 @@ def validate(self, data):
 
 class PeerFacilitySyncJobValidator(PeerSyncJobValidator):
     username = serializers.CharField()
-    password = serializers.CharField()
+    password = serializers.CharField(required=False)
```

Comment:
Could set the default here as `"NOT_SPECIFIED"`.

I was also surprised to discover that we don't have a constant in Python code anywhere for the `"NOT_SPECIFIED"` password constant. I guess a consistently named constant vs a consistently named string is about the same?

Reply:
Done.
I'm also very surprised for the lack of constant for `NOT_SPECIFIED` We're using it both in JS and Python but it's not defined. I thought of doing it here but I didn't want to taint this PR with that refactoring. I'll do it in a different PR after this work is finished. There are many different parts of the application affected